js/views/onboardingModal.js

```diff
@@ -351,6 +351,10 @@ module.exports = baseModal.extend({
                 console.log(errorThrown);
               }
             });
+          } else {
```

Comment:
Still don't understand why the settings fails and the socket does not. There must be more pieces to the puzzle here than I'm aware of.

Anyhow, is this ready for merge?